AMW_angular/io/src/app/deployment/deployments.component.ts

```diff
@@ -665,4 +652,12 @@ export class DeploymentsComponent implements OnInit {
     this.isLoading = false;
   }
 
+  private goTo(destination: string) {
```

Comment:
goTo is a very generic name, can we call this updateFiltersInURL or something more descriptive?